bin/get_latest_majors.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+
+lines = [l.strip() for l in sys.stdin]
+splits = [l.split(".") for l in lines]
```

Comment:
I would add some sort of comment for this script, or maybe rename it to `get_latest_majors_for_ci`. When people will browse the repository and look into the bin folder before reading the workflows, it might not be clear what this script does or that it's used on CI.

Reply:
You are probably right about this.